content/community/events/cloud-native-hamburg-2019-08.md

```diff
@@ -1,6 +1,6 @@
 ---
 event: Hamburg Kubernetes/Cloud-Native Meetup
-topic: Databases on K8s Using a Custom Operator & Scalable Graph Processing on K8s
+topic: Databases on Kubernetes Using a Custom Operator & Scalable Graph Processing on K8s
```

Comment:
```suggestion
topic: Databases on K8s Using a Custom Operator & Scalable Graph Processing on K8s
```
Nit: Let's not change event titles -- there's also a "K8s" at the end of the title.